src/test/java/org/fundacionjala/pivotal/steps/story/CreateStorySteps.java

```diff
@@ -1,9 +1,10 @@
 package org.fundacionjala.pivotal.steps.story;
 
+import cucumber.api.PendingException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'cucumber.api.PendingException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)